test/app-luatest/protobuf_I32_test.lua

```diff
@@ -0,0 +1,276 @@
+local t = require('luatest')
```

Comment:
Nit: I'm not sure that grouping tests by a wire type is the best option.

This way we repeat the same scenarios (default value, positive/negative value, mix/max values, positive/negative out of range values, improver value type, improper value) in the same file several times, but, at the same time, repeat them in different files.

I'm not against any way of grouping, in fact. I'm not sure here, just thoughts.

If you're going to think about declarative (parametrized) test cases, I propose to think about splitting to files too.

Reply:
I tried to group tests inside the existing files, but it was pointless due to similarity of the tests in `VARINT`, `I32` and `I64`. So I made one file with parametrized tests for all numeric protobuf types. If it will be better to split this file I will do it, but it turned out to be smaller than each previous file separately.